extensions/wikia/AdEngine/js/spec/video/playerTracker.spec.js

```diff
@@ -185,6 +185,21 @@ describe('ext.wikia.adEngine.video.player.playerTracker', function () {
 		expect(getTrackedValue('price')).toEqual('1.20');
 	});
 
+	it('Track data with Beachfront data for beachfront ad product', function () {
+		tracker.track({
+			adProduct: 'beachfront',
+			slotName: 'TOP_LEADERBOARD',
+			bid: {
+				bidderCode: 'beachfront',
+				creative_id: 'w1k14',
+				cpm: 456
```

Comment:
I'm not sure how this cpm get calculated to `1.20`. Are you sure this test passes?

Reply:
Fixed hardcoded value in tests.